prow/pjutil/pjutil_test.go

```diff
@@ -1211,6 +1212,28 @@ func TestSpecFromJobBase(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			name: "Verify DecorateExtraRefs",
+			jobBase: config.JobBase{
+				UtilityConfig: config.UtilityConfig{
+					DecorationConfig: &prowapi.DecorationConfig{
+						BloblessFetch: &trueVar,
+					},
+					ExtraRefs: []prowapi.Refs{{
```

Comment:
How about adding another ExtraRef case with `BloblessFetch: true` here to check that it stays `true` after defaulting?

Another separate case you could add is what happens if BloblessFetch is not specified in ExtraRefs (i.e., does it pick up the `true` setting from DecorationConfig?).

Reply:
I've written three tests cases now, demonstrating true and false defaults, and preservation of values without defaults. I hope this clarifies the meaning of my statement in the description.

Reply:
I understand the test cases well enough, but your statement in the description is still hard to understand for me. Maybe it's just me, but I don't have a strong opinion on this, so LGTM.

Reply:
I've reworded the note in the description. Let me know if you have suggestions.